runtime/drivers/duckdb/config.go

```diff
@@ -50,9 +58,11 @@ func newConfig(dsn string) (*config, error) {
 
 	// Return config
 	cfg := &config{
-		DSN:        dsn,
-		PoolSize:   poolSize,
-		DBFilePath: uri.Path,
+		DSN:          dsn,
+		PoolSize:     poolSize,
+		DBFilePath:   uri.Path,
+		Activity:     client,
+		ActivityDims: activityDims,
```

Comment:
Feels like this doesn't belong on the `config` struct but on the main handle, alongside the `logger`